docs/topics/js/js-modules.md

```diff
@@ -33,22 +33,22 @@ kotlin {
 Webpack provides two different "flavors" of CommonJS, `commonjs` and `commonjs2`, which affect the way your declarations
 are made available. While in most cases, you probably want `commonjs2`, which adds the `module.exports` syntax to the
 generated library, you can also opt for the "pure" `commonjs` option, which implements the CommonJS specification exactly.
-To learn more about the difference between `commonjs` and `commonjs2`, check [here](https://github.com/webpack/webpack/issues/1114).
+To learn more about the difference between `commonjs` and `commonjs2`, see the [Webpack repository](https://github.com/webpack/webpack/issues/1114).
 
 ## JavaScript libraries and Node.js files
 
-If you are creating a library that will be consumed from JavaScript or a Node.js file, and want to use a different module
+If you are creating a library to be consumed from JavaScript or a Node.js file, and want to use a different module
 system, the instructions are slightly different.
 
 ### Choose the target module system
 
-To select module kind, set the `moduleKind` compiler option in the Gradle build script.
+To select module kind, set the `moduleKind` compiler option in the Gradle build script:
```

Comment:
I would suggest changing it to: "To select the target module system" to be consistent with the title and the previous description.